docs/help-topics/stonesoup-import/stonesoup-import.yml

```diff
@@ -41,16 +41,21 @@
   content: |-
     We analyze your code, then define and apply deployment settings for each of your application components. Remember, a “component” is an image that we build from source code in a repository. An application is a collection of these components working together.
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too. 
 
-    - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
-    - **Route**: Enter the public URL for your published application.
-    - **Environment variables**: Define your environment variables to customize how we deploy your application. Usually, you define these in a devfile or Dockerfile. If your application doesn’t contain either of these file types, we use an application analyzer to extract the necessary information from your code.
-    - **Runtime**: Did we detect the right runtime? If so, you don’t have to do anything with this field. If you want to change your runtime, click the **Runtime** menu and make a selection.
-    - **Send pull request**: Select this option to create a custom pipeline in your Git repository.
+    - Before you expand all of your configuration options, check the following:
```

Comment:
Should we assume that the first thing they did was open the Learn more content? What if they clicked on stuff to explore the page, and then decided they needed more help and looked at this? Their page state may not match what you describe here and later down.

Reply:
you're right: users start to explore, then encounter something that prompts them to look for more info. i wrote this help from the top down, which is the usual way when we can't be sure where/what a user will check out first. you think context-specific tooltips/hints/popovers/text areas could be better here @mattreid?

Reply:
I'm fine with having supplemental information in Learn more content, although I think it's a great question on what level of detail and content belongs where. In some cases we have default text in a field, help text under that field, a tooltip next to the field label, and some additional info in Learn more.  

My comment was just wondering if writing it assuming they're following from a default state and walking them through the process as if they haven't already explored the page also works well for the user who thought they were fine, started the process, got hung up and is hoping this content can answer their question. It seems like it might make it harder to skim for whatever help they were looking for.

Reply:
It's not a major concern on my part. As long as it's structured and scannable, I'm happy. Looking at the Add component learn more content, looks like it fits with the approach there. We can close this thread out 👍 .